akka-operator/values.yaml

```diff
@@ -9,7 +9,7 @@ image:
   tag: "1.1.18"
 ubbagent:
   image:
-    repository: gcr.io/lightbend-public/akka-cloud-platform/ubbagent
+    repository: gcr.io/cloud-marketplace/lightbend-public/akka-cloud-platform/ubbagent
```

Comment:
`lightbend-public` is not publicly accessible since is a private repository, but all the GKE clusters have access to marketplace